tests/tests.js

```diff
@@ -25,15 +13,17 @@ var assert = chai.assert;
 var testUtils = require('./test-utils');
 var config = require('../lib/config');
 var www = require('../bin/www');
+var models = require('../models');
 
 var notFoundURL = testUtils.appHost + '/page-that-will-never-exist';
 
-// Announce that automated tests are being run just in case other parts of the
-// application want to behave differently
-process.env.RUNNING_TESTS = true;
-
-
 describe('Discord Tests', function() {
+    this.timeout(5000); // Travis can be a little slow
```

Comment:
Hmm, can this be set higher? `this` is undefined outside of the `describe` callback.